src/plugins/news/components/headline-fragments-card/index.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Card, CardBody, CardHeader } from 'reactstrap';
+import classNames from 'classnames';
+import { Col, Row } from 'reactstrap';
+import { useFieldArray } from 'react-final-form-arrays';
+import { SelectField, TextField } from '@triniti/cms/components/index.js';
+
+const OPTIONS_STYLES = [
+  { label: 'UPPERCASE', value: 'uppercase' },
+  { label: 'TitleCase', value: 'titlecase' },
+  { label: 'no styling', value: 'none' },
+];
+const OPTIONS_SIZE = [
+  { label: 'XL', value: 1 },
+  { label: 'L', value: 2 },
+  { label: 'M', value: 3 },
+  { label: 'S', value: 4 },
+  { label: 'XS', value: 5 },
+  { label: 'XXS', value: 6 },
+];
+
+export default function HeadlineFragmentsField(props) {
+  const { pbjName, groupClassName = '' } = props;
```

Comment:
@rsumilang is this working?  seems like we're missing a lot to make this work

Reply:
![CleanShot 2024-07-17 at 08 45 23](https://github.com/user-attachments/assets/358d1809-70f1-464e-b61c-a6c4dc005072)

 Seems like it

Reply:
@rsumilang what about with new articles?  also, try not setting size/style or text on some and save.  note that pbjName refers to the schema's name, whereas "name" is for final form.  e.g. https://github.com/triniti/cms-js/blob/v2.x/src/plugins/curator/components/widget-screen/SearchArticlesRequestFields.js#L26

Reply:
@rsumilang we don't have a nested pbj (like poll answers or widget datasource), we have three separate fields that are arrays that when managed should always be the same length/count.